api/v1alpha1/jobset_webhook.go

```diff
@@ -54,6 +56,13 @@ func (r *JobSet) Default() {
 		}
 	}
 
+	// Default success policy to all jobs.
+	if r.Spec.SuccessPolicy == nil {
+		r.Spec.SuccessPolicy = &SuccessPolicy{
+			Operator:    OperatorAll,
+			JobSelector: &metav1.LabelSelector{},
+		}
+	}
```

Comment:
If only `successPolicy.jobSelector` is unset (only `successPolicy.operator` is set), wouldn't we need to initialize `successPolicy.jobSelector` as empty?

Reply:
The webhook defaults it to an empty label selector if it is unset.

Reply:
We decided to go another direction with the success policy API (see [here](https://github.com/kubernetes-sigs/jobset/pull/124#discussion_r1195807489)) so this comment no longer applies. This PR is ready for another review now though if you want to take a look.